vm/src/compile.rs

```diff
@@ -644,6 +657,10 @@ impl Compiler {
             });
         }
 
+        self.emit(Instruction::LoadConst {
```

Comment:
Modify the code here something like this: ```rust
if let Some(doc_string) = doc_str {
  self.emit(LoadConst { doc_string });
  self.emit(StoreAttr("__doc__"));
}
// else -> no code emitted!
```

Reply:
Btw, micropython is a good source of inspiration for these kind of things: https://github.com/micropython/micropython/blob/master/py/compile.c#L3015